ImperatorToCK3Tests/MapperTests/TraitMapper/TraitMappingTests.cpp

```diff
@@ -0,0 +1,49 @@
+#include "../../ImperatorToCK3/Source/Mappers/TraitMapper/TraitMapping.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+
+TEST(Mappers_TraitMappingTests, ck3TraitDefaultsToBlank)
```

Comment:
Is an empty string really what you want in this case? I would expect either a nullopt, or if its sufficiently unexpected, a thrown error.

If empty string really is what you want, you should include tests in the mapper and in character to demonstrate how it works.

Reply:
You're right, changing to nullopt.